compiler/src/codegen.rs

```diff
@@ -588,9 +594,14 @@ impl<'a> ServiceGen<'a> {
         );
 
         let has_stream_method = self.has_stream_method();
+        let has_normal_method = self.has_normal_method();
```

Comment:
There is the same problem with generated code in sync-mode. I would appreciate it if you can fix it as well.

Reply:
Added.